config/initializers/git_sha.rb

```diff
@@ -0,0 +1,25 @@
+GIT_SHA =
+  if Rails.env.production? && File.exist?('/opt/laevigata/revisions.log')
+    `tail -1 /opt/laevigata/revisions.log`.chomp.split(" ")[3].gsub(/\)$/, '')
```

Comment:
I think you mean `/opt/mahonia` here

Reply:
Should all of these be an ENV variable, perhaps?